docker/docker-compose.yml

```diff
@@ -103,7 +103,7 @@ services:
     environment:
       - "IML_CA_PATH=/var/lib/chroma/authority.crt"
   device-aggregator:
-    image: "imlteam/device-aggregator:5.1"
+    image: "imlteam/device-aggregator:latest"
```

Comment:
`:latest` [is considered](https://www.reddit.com/r/docker/comments/87io7e/whats_wrong_with_the_docker_latest_tag/) harmful. But I don't know what would be the best here.

Reply:
yeah, I'm hopeful this is short-lived as it will go away with the device-scanner integration into the plugin architecture.

For now, we push infrequently enough (and want to test all the changes) that I think latest is ok.